src/utils/index.js

```diff
@@ -1,6 +1,10 @@
 import { get } from 'lodash-es';
 import BigNumber from 'bignumber.js';
 import i18n from './i18nHelper';
+// eslint-disable-next-line import/no-cycle
+import Backend from './backend';
+// eslint-disable-next-line import/no-cycle
+import store from '../store/index';
```

Comment:
Utils shouldn't import any 'heavy' modules, you can get rid of cycle dependencies by using `callWithAuth` from the store. Let's make `callWithAuth` to return the backend reponse only if sdk wallet available.